### PR TITLE
move(docs/contrib-guidelines): Add contrib guidelines to Maestro

### PR DESCRIPTION
**Pull request by nuclearcat**
Reformat a bit, removing obsolete bits. Guidelines will need further work and improvements.

### CODE REVIEW

File: kernelci.org/content/en/contacts.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Contact us"
+date: 2024-09-12
+description: "How to contact us"
+weight: 1
+---
+
+> KernelCI community contacts are available on the [KernelCI website](https://kernelci.org/community-contact//).
```
Comment (patersonc) [inline kernelci.org/content/en/contacts.md:8]: nitpick: Only needs a single / on the end of the URL.

File: kernelci.org/content/en/maestro/contrib.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+   1. If the PR is expected to produce build/test results
+   check [staging viewer](https://staging.kernelci.org:9000/viewer) and make sure these are mentioned in the PR comment
+      1. Build artifacts including logs are not kept permanently, so it's generally recommended to put them in a place that'd make them last if you want them to be part of the PR. Good way to do that seem to be:
+         * Putting important information such as log fragments in the PR comments
+         * Using services like [pastebin](https://pastebin.com/) to store data important for the PR (e.g. full logs) and pasting the links.
+   2. If the results are not visible on staging and you think they should be, mention it in the comments, too
+   3. If there is specific way to verify PR is working as expected, mention it in the comments
+3. Make sure that reviewers' comments and questions are addressed
+   1. When there are comments unanswered for more than 1 month the PR will be closed
+4. In case there is a need to consult the PR with KernelCI maintainers join the open hours
```
Comment (patersonc) [inline kernelci.org/content/en/maestro/contrib.md:31]: Shall we link to https://docs.kernelci.org/org/#open-hours here?

**Comment by patersonc** [comment]
@nuclearcat do you agree with my suggestions? I'm happy to merge either way tbh, let me know.

File: kernelci.org/content/en/contacts.md (hunk 1 above)
Comment (nuclearcat) [reply kernelci.org/content/en/contacts.md]: fixed

File: kernelci.org/content/en/maestro/contrib.md (hunk 2 above)
Comment (nuclearcat) [reply kernelci.org/content/en/maestro/contrib.md]: fixed

**Comment by nuclearcat** [comment]
Sorry for delay, was a bit drowned in other tasks, desperate to get results :) now returning to normal mode.
I edited all suggestions.

File: kernelci.org/content/en/maestro/contrib.md (hunk 2 above)
Comment (patersonc) [reply kernelci.org/content/en/maestro/contrib.md]: Sorry @nuclearcat , I think you may have put the wrong link in here.

Comment (nuclearcat) [reply kernelci.org/content/en/maestro/contrib.md]: Can you check now?

**Review by patersonc — APPROVED** [review]

**Pull request merged**